### PR TITLE
[8.11]Findings page EUI updates

### PR DESCRIPTION
**Pull request by benironside**
Addresses #4060 (parallel issue required for serverless)

Previews:

[Findings page (CSPM)](https://security-docs_4084.docs-preview.app.elstc.co/guide/en/security/master/cspm-findings-page.html)
[Findings page (CNVM)](https://security-docs_4084.docs-preview.app.elstc.co/guide/en/security/master/vuln-management-findings.html)
[Security UI overview, Findings section](https://security-docs_4084.docs-preview.app.elstc.co/guide/en/security/master/es-ui-overview.html)

### CODE REVIEW

**Review by tinnytintin10 — APPROVED** [review]
LGTM

**Review by natasha-moore-elastic — APPROVED** [review]
One question, otherwise LGTM!

File: docs/cloud-native-security/cspm-findings.asciidoc [hunk 1]
```diff
@@ -32,6 +32,15 @@ You can filter findings data in two ways:
 * *The KQL search bar*: For example, search for `result.evaluation : failed` to view all failed findings.
 * *In-table value filters*: Hover over a finding to display available inline actions. Use the Filter In (plus) and Filter Out (minus) buttons.
 
+[discrete]
+[[cspm-customize-the-findings-table]]
+=== Customize the Findings table
+Use the toolbar buttons in the upper-left of the Findings table to customize the columns you want displayed:
+
+* **Columns**: Reorder the columns.
```
Comment (benironside) [inline docs/cloud-native-security/cspm-findings.asciidoc:40]:
```suggestion
* **Columns**: Select which columns to display.
```

File: docs/cloud-native-security/kspm-findings.asciidoc [hunk 2]
```diff
@@ -32,6 +32,15 @@ You can filter findings data in two ways:
 * *The KQL search bar*: For example, search for `result.evaluation : failed` to view all failed findings.
 * *In-table value filters*: Hover over a finding to display available inline actions. Use the Filter In (plus) and Filter Out (minus) buttons.
 
+[discrete]
+[[kspm-customize-the-findings-table]]
+=== Customize the Findings table
+Use the toolbar buttons in the upper-left of the Findings table to customize the columns you want displayed:
+
+* **Columns**: Reorder the columns.
```
Comment (benironside) [inline docs/cloud-native-security/kspm-findings.asciidoc:40]:
```suggestion
* **Columns**: Select which columns to display.
```

File: docs/cloud-native-security/vuln-management-findings.asciidoc [hunk 3]
```diff
@@ -36,6 +36,14 @@ Click a vulnerability to open the vulnerability details flyout. This flyout incl
 
 When you open the vulnerability details flyout, it defaults to the *Overview* tab, which highlights key information. To view every field present in the vulnerability document, select the *Table* or *JSON* tabs.
 
+[discrete]
+[[cnvm-customize-the-findings-table]]
+=== Customize the Findings table
+Use the toolbar buttons in the upper-left of the Findings table to customize the columns you want displayed:
+
+* **Columns**: Reorder the columns.
```
Comment (benironside) [inline docs/cloud-native-security/vuln-management-findings.asciidoc:44]:
```suggestion
* **Columns**: Select which columns to display.
```

**Review by nastasha-solomon — APPROVED** [review]
Changes lgtm!

**Review by jmikell821 — APPROVED** [review]

**Pull request merged**